src/Design.cc

```diff
@@ -98,6 +98,249 @@ void Design::readDef(const std::string& file_name,
                child);
 }
 
+
+
+odb::dbITerm* Design::staToDBPin(const sta::Pin *pin) {
```

Comment:
This isn't a good API as iterm may be null - a Pin is either an iterm or a bterm not always an iterm.